components/dashboard/Menu.js

```diff
@@ -0,0 +1,373 @@
+          expanded={expandedSection === 'FISCAL_HOST_SETTINGS'}
+          subMenu={
+            <React.Fragment>
+              <MenuLink isSub collective={collective} section={FISCAL_HOST_SECTIONS.FISCAL_HOSTING} />
+              <MenuLink isSub collective={collective} section={FISCAL_HOST_SECTIONS.INVOICES_RECEIPTS} />
+              <MenuLink isSub collective={collective} section={FISCAL_HOST_SECTIONS.RECEIVING_MONEY} />
+              <MenuLink isSub collective={collective} section={FISCAL_HOST_SECTIONS.SENDING_MONEY} />
+              <MenuLink
+                isSub
+                collective={collective}
+                section={FISCAL_HOST_SECTIONS.HOST_VIRTUAL_CARDS_SETTINGS}
+                if={hasFeature(collective, FEATURES.VIRTUAL_CARDS)}
+              />
+              <MenuLink
+                isSub
+                collective={collective}
+                section={FISCAL_HOST_SECTIONS.POLICIES}
+                if={isOneOfTypes(collective, [USER, ORGANIZATION])}
+              />
+            </React.Fragment>
+          }
+        >
+          <FormattedMessage id="AdminPanel.FiscalHostSettings" defaultMessage="Fiscal Host Settings" />
+        </MenuLink>
+      </MenuGroup>
+
+      {/** User/org/collective/event/project dashbord */}
+      <MenuGroup>
+        <MenuSectionHeader>
+          {isType(collective, ORGANIZATION) ? (
+            <FormattedMessage id="OrganizationDashboard" defaultMessage="Organization Dashboard" />
+          ) : isType(collective, USER) ? (
+            <FormattedMessage id="UserDashboard" defaultMessage="User Dashboard" />
+          ) : isType(collective, COLLECTIVE) ? (
+            <FormattedMessage id="CollectiveDashboard" defaultMessage="Collective Dashboard" />
+          ) : (
+            <FormattedMessage id="Dashboard" defaultMessage="Dashboard" />
+          )}
+        </MenuSectionHeader>
+
+        <MenuLink
+          collective={collective}
+          section={COLLECTIVE_SECTIONS.MANAGE_CONTRIBUTIONS}
+          icon={<Coin size={16} />}
+          setExpanded={setExpanded}
+        />
+        <MenuLink
+          setExpanded={setExpanded}
+          collective={collective}
+          section={ORG_BUDGET_SECTIONS.FINANCIAL_CONTRIBUTIONS}
+          icon={<Coin size={16} />}
+          if={isSelfHostedAccount(collective) && !isAccountantOnly && isType(collective, COLLECTIVE)}
+        />
+        <MenuLink
+          setExpanded={setExpanded}
+          collective={collective}
+          section={COLLECTIVE_SECTIONS.EXPENSES}
+          icon={<Receipt size={16} />}
+        />
+        <MenuLink
+          setExpanded={setExpanded}
+          collective={collective}
+          section={COLLECTIVE_SECTIONS.TRANSACTIONS}
+          icon={<Transfer size={16} />}
+        />
+        <MenuLink
+          collective={collective}
+          icon={<Cog size={16} />}
+          if={!isHost}
+          setExpanded={setExpanded}
+          section="COLL_SETTINGS"
```

Comment:
strange name

Reply:
Could you look into this or explain the cryptic name?

Reply:
Pushed a fix to rename it to "SETTINGS".

It's a bit of a "pseudo" section in the sense that it only acts as a submenu with the section name only there to provide a way to keep track of what submenu/section is currently expanded, this could be refactored or simplified a bit.